core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -849,7 +856,18 @@ $(document).ready(function () {
             {data: "partitionId", name: "Index"},
             {data : "taskId", name: "ID"},
             {data : "attempt", name: "Attempt"},
-            {data : "status", name: "Status"},
+            {
+              data : (row, _ignored_type) => {
+                if (taskThreadDumpEnabled && row.status === "RUNNING") {
```

Comment:
To confirm, this will show up only in live UI (due to `StagesTab,threadDumpEnabled`), right ?.

Reply:
Yes, Live only,  SC is required for display.